datumaro/components/media.py

```diff
@@ -58,21 +88,32 @@ def ext(self) -> str:
         return osp.splitext(osp.basename(self.path))[1]
 
     @property
-    def is_encrypted(self) -> bool:
-        return not self._crypter.is_null_crypter
+    def data(self) -> Any:
+        data = None
+        if os.path.exists(self.path):
+            with open(self.path, "wb") as f:
+                data = f.read()
+        return data
 
-    def set_crypter(self, crypter: Crypter):
-        self._crypter = crypter
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}(path={self._path})"
 
-    def __eq__(self, other: object) -> bool:
-        # We need to compare exactly with this type
-        if type(other) is not __class__:  # pylint: disable=unidiomatic-typecheck
-            return False
-        return self._path == other._path
+
+class MediaElementFromDataMixin:
+    def __init__(self, data: Any, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, data: Union[Callable[[], Any], Any], *args, **kwargs):
```

I think we can replace `Any` with the union of data types used in all media types. It might be declared in the somewhere above.

Reply:
https://github.com/openvinotoolkit/datumaro/pull/911/commits/f11cb163936e0cd9b154cb2f4fc2f602b12581db